pkgs/applications/video/olive-editor/default.nix

```
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, pkgconfig, which, qt5, frei0r, opencolorio, ffmpeg-full  }:
+
+stdenv.mkDerivation
+{
+  pname = "olive-editor";
+
+  version = "2019-03-18";
+
+  src = fetchFromGitHub {
+    owner = "olive-editor";
+    repo = "olive";
+    rev = "d153e4d7471122987098087a203323003ae8a128";
+    sha256 = "1gxpz6jg6dqjmvqkk0m9g4pz30lklwx51d73gka404a3w1j5wna6";
+  };
+
+  nativeBuildInputs = [ 
+    pkgconfig 
+    which 
+    qt5.qmake
+  ];
+
+  buildInputs = [
+    frei0r
+    ffmpeg-full
+    opencolorio
+  ] ++
+  (with qt5; [
+    qtbase
+    qtmultimedia
+    qtmultimedia.dev
+  ]);
+
+  meta = with stdenv.lib; {
+    description = "Professional open-source NLE video editor";
+    homepage = "https://www.olivevideoeditor.org/";
+    downloadPage = "https://www.olivevideoeditor.org/download.php";
+    license = licenses.gpl3;
+    maintainers = [ maintainers.balsoft ];
+    platforms = platforms.linux;
```

```suggestion
    platforms = platforms.unix;
```
This has macos support

---

Regarding mac os support, you may need to condition what buildInputs, etc are added per platform.

`stdenv.lib.optionals stdenv.isDarwin` or `stdenv.isLinux` for linux would be useful
> stdenv.lib.optionals
> https://github.com/NixOS/nixpkgs/blob/master/lib/lists.nix#L243

Also here are the [libs for darwin](https://github.com/olive-editor/olive/blob/54c03efd5481fb596f7d0ecca3badab6376fe16e/olive.pro#L332)

---

Not sure if it'll work, but I've passed CoreFoundation through.

---

Neither have I, if we need someone to test/review it I can find someone for you.